webapp/portlet/src/main/webapp/vue-apps/common/components/RichEditor.vue

```diff
@@ -202,7 +202,7 @@ export default {
       if (this.supportsOembed) {
         this.setOembedParams({
           default_title: this.getContentToSave(this.inputVal),
-          comment: this.getContentNoEmbed(this.inputVal),
+          comment: this.getContentToSave(this.inputVal),
```

Comment:
Why this change, this may cause problems in notifications by example. a very large test has to be made to verify that this will not impact other modules/expected behaviors.
If the Kudos module needs the comment to has the list of mentioned users in this property, we may specify this behavior for kudos only by a property to no apply a widely impacting modification.
Please double check the need of modifying this and if yes, can you please provide further information.